vuepress/docs/next/tutorials/consume/mt-cds.md

```diff
@@ -0,0 +1,285 @@
+| YOUR-APPNAME | Your app name, or the name of the entando-de-app deployment
+| YOUR-HOSTNAME | Host name (e.g., k8s-entando.org)
+| YOUR-NAMESPACE | Namespace used for the multitenancy application
+| YOUR-PRIMARY-KC-SECRET | Name of Keycloak secret for the primary tenant
+| YOUR-PRIMARY-CDS-DEPLOYMENT | CDS deployment for the primary tenant
+| YOUR-PRIMARY-PVC | Name of PVC claim of the primary tenant
+| YOUR-PRIMARY-CDS-SERVICE | The CDS service for the primary tenant
+| YOUR-PRIMARY-CDS-DEPLOYMENT | The CDS deployment of the primary tenant
+| YOUR-PRIMARY-CDS-INGRESS | The CDS ingress for the primary tenant
+| YOUR-PRIMARY-TENANT | Name of the primary tenant and primary tenant code 
+| YOUR-PUBLIC-KEY | Keycloak RSA generated string for your realm
+
+### Keycloak Descriptor YAML
+1. Go to Keycloak admin console and log in
+2. From the left sidebar, `Realm Settings` → `Keys`. Click on `Public Key` for `rsa-generated` provider and copy the content.
+
+3. Create a descriptor using this example. Replace `YOUR-PUBLIC-KEY` with the result from the previous step, following the `---BEGIN ... END---\n` format shown below:
+
+``` yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: YOUR-PRIMARY-KC-SECRET
+  namespace: YOUR-NAMESPACE
+type: Opaque
+stringData:
+  KC_PUBLIC_KEY: "-----BEGIN PUBLIC KEY-----\nYOUR-PUBLIC-KEY\n-----END PUBLIC KEY-----\n"
+```
+
+### Persistent Volume Claim Descriptor
+This descriptor should provide specifications for the persistent volume claim, including accessModes and resources storage limits. 
+
+``` yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    deployment: YOUR-PRIMARY-CDS-DEPLOYMENT
+  name: YOUR-PRIMARY-PVC
+  namespace: YOUR-NAMESPACE
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    limits:
+      storage: 1Gi
+    requests:
+      storage: 1Gi
+```
+
+### Service and Deployment Descriptor
+Create the service and deployment descriptor YAML with the specifications required for your environment in a K3s implementation. For convenience, the service and deployment has been combined into a single descriptor `YOUR-PRIMARY-CDS-DEPLOYMENT.yaml`, but this is a matter of preference.
+
+``` yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: YOUR-PRIMARY-CDS-SERVICE
+  namespace: YOUR-NAMESPACE
+  labels:
+    app: YOUR-PRIMARY-CDS-SERVICE
+spec:
+  ports:
+    - port: 8080
+      name: internal-port
+      protocol: TCP
+      targetPort: 8080
+    - port: 8081
+      name: public-port
+      protocol: TCP
+      targetPort: 8081
+  selector:
+    app: YOUR-PRIMARY-CDS-DEPLOYMENT
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: YOUR-PRIMARY-CDS-DEPLOYMENT
+  namespace: YOUR-NAMESPACE
+  labels:
+    app: YOUR-PRIMARY-CDS-DEPLOYMENT
+spec:
+  selector:
+    matchLabels:
+      app: YOUR-PRIMARY-CDS-DEPLOYMENT
+  template:
+    metadata:
+      labels:
+        app: YOUR-PRIMARY-CDS-DEPLOYMENT
+    spec:
+      initContainers:
+	        - name: init-cds-data
```

Comment:
@jyunmitch I guess the suggestion made with the GitHub UI has broke the indentation, we should remove the part in red here (it seems to be equivalent to one tabulation